org-eclipse-che-jdt-ls-extension/jdt.ls.extension.core/plugin.xml

```diff
@@ -12,6 +12,9 @@
             <command id="che.jdt.ls.extension.resolveClasspath"/>
             <command id="che.jdt.ls.extension.outputDir"/>
             <command id="org.eclipse.che.jdt.ls.extension.filestructure"/>
+            <command
```

Comment:
I don't see any reasons not to follow to format above:
`<command id="che.jdt.ls.extension.findImplementors"/>`

Reply:
Actually i added this command Eclipse UI interface for "plugin.xml" files and it made this format itself :) i haven't even paid attention to it. Definitely will fix it :)